packages/cli/src/swagger/specGenerator3.ts

```diff
@@ -639,7 +652,7 @@ export class SpecGenerator3 extends SpecGenerator {
     if (types.size === 1) {
       const type = types.values().next().value;
       const nullable = enumType.enums.includes(null) ? true : false;
-      return { type, enum: enumType.enums.map(member => (member === null ? null : String(member))), nullable };
+      return { type, enum: enumType.enums.map(member => (member === null ? null : getValue(type, member))), nullable };
```

Comment:
```suggestion
      return { type, enum: enumType.enums.map(member => getValue(type, member)), nullable };
```

Reply:
Hm my main objective was to fix the issue - any reason for putting the `null` handling in the `getValue`? We can do it, just wanted to know "why" (and why in the context of that issue).

Reply:
I think it's better to handle that functionality in one place if possible and move the conditional out of the already pretty lone one-liner. Not a cause for rejection, but it'd be preferable.